ovh/resource_dedicated_server_install_task.go

```diff
@@ -105,17 +93,25 @@ func resourceDedicatedServerInstallTask() *schema.Resource {
 							ForceNew:    true,
 							Description: "",
 						},
-						"ssh_key_name": {
+					},
+				},
+			},
+			"user_metadata": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 10,
```

Comment:
```suggestion
				MaxItems: 128,
```
10 is the number of key/value couples for the answer of an input of type keyValue